cloudsync/sync/manager.py

```diff
@@ -331,7 +331,7 @@ def sync(self, sync: SyncEntry) -> bool:  # pylint: disable=too-many-branches,to
 
         ordered = sorted((LOCAL, REMOTE), key=lambda e: sync[e].changed or 0)
 
-        something_got_done = False
+        something_got_done = True
```

Comment:
i would rather default to false, and then tag the cases as true...

Reply:
from our talk... we should set the flag to False if get_latest did nothing, and True if it did something in the cloud (which we can, as a hack, assume is the remote provider)

Reply:
resolved per offline conversation -- the idea here is to proactively (and safely) clear backoff -- during long walks we are likely to process many NOOP events which do not currently clear backoff, resulting in degraded performance for a potentially long period of time